### ISSUE TITLE
Select with Default Blank selected or List Type 

### ISSUE THREAD

**Issue by packytagliaferro** [Laravel-Backpack/CRUD#1043]
Right now I have 2 Models: User and Company. Basically when I click on Users I just want to see what company they belong to and when I click on a Company I want to see all the users that belong to the company. 

I thought I could just do a Select2 type and disable it, which does work kind of. The issue is when add a company, if pre selects a user to add (the first user is defaulted as selected) when I dont want to be able to add any users. 

In short, this is for the Super Admin and I dont want them to select what user goes to what company, that is for the Company to do. All I want is to display the company on the user form and and users on the the company form. 

I also tried a text field, but that just displays the ID which doesnt help.

**Comment by lloy0076** [comment]
You could use a text field with an accessor, couldn't you?

**Comment by packytagliaferro** [comment]
@lloy0076 from what I understand if I set an accessor on company ID to return the name of the company every time the app attempts to get the company->id it will return the name which wont work.

**Comment by lloy0076** [comment]
That said, for your company name issue, you'd make up a field name and an accessor:

```
public function getFooFieldAttribute()
{
    return "something";
}
```

And then setup a text field in the controller with `name => 'fooField'`.

It will certainly work in the edit/create forms but it might not behave on the initial datatable listing. There I think you'd need to add it to `$fillable` and it's possible you'll hit the AJAX issues that should go away in a soon to be released release/update.

I think you'd need to store the actual ID in a hidden field to help the rest of the CRUD default code to work properly.

**Comment by packytagliaferro** [comment]
@lloy0076 I used your idea and did it in a `custom_html` type. Maybe this could be helpful to someone else in the docs to export custom data that doesnt need to be updated?

On My Company Model
```

public function getCrudUsersAttribute()
		{
			$managers = '<h2>Market Managers</h2>';

			foreach( $this->users as $user ){
				$managers .=  $user->username . ' - ' .$user->email .  '<br>';
			}

			return $managers;
		}
```

in my CompanyCrudController

```
[
                'name' => 'crud_users',
                'readonly'=>'readonly',
                'type' => 'custom_html',
            ],
```
